re-former/app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+	validates :username, presence:true
```

Comment:
- We can validate multiple attributes in the same line if they all share the same validator
`validates :username, :email, :password, presence: true`